src/Components/Auction/RegistrationFlowModal.tsx

```diff
@@ -0,0 +1,132 @@
+import { Box, Button, Serif } from "@artsy/palette"
+import { RegistrationFlowModalQuery } from "__generated__/RegistrationFlowModalQuery.graphql"
+import { useSystemContext } from "Artsy"
+import { ConditionsOfSaleCheckbox } from "Components/Auction/ConditionsOfSaleCheckbox"
+import { Modal } from "Components/Modal/Modal"
```

Comment:
@dblandin Hey you should be good to go using the new Palette Modal here. Let me know if you encounter any issues.

Reply:
Happy to look at this with you but I had some trouble with it so stuck with the reaction modal for now!